quickjs.h

```diff
@@ -335,6 +335,12 @@ JS_EXTERN void JS_AddIntrinsicBigInt(JSContext *ctx);
 JS_EXTERN void JS_AddIntrinsicWeakRef(JSContext *ctx);
 JS_EXTERN void JS_AddPerformance(JSContext *ctx);
 
+/* for equality comparisons and sameness */
+JS_EXTERN JS_BOOL JS_IsLooselyEqual(JSContext *ctx, JSValue op1, JSValue op2);
+JS_EXTERN JS_BOOL JS_IsStrictlyEqual(JSContext *ctx, JSValue op1, JSValue op2);
+JS_EXTERN JS_BOOL JS_IsSameValue(JSContext *ctx, JSValue op1, JSValue op2);
+JS_EXTERN JS_BOOL JS_IsSameValueZero(JSContext *ctx, JSValue op1, JSValue op2);
```

Comment:
- I'd name the first two JS_IsEqual and JS_IsStrictEqual

- can you add a comment explaining the difference between IsSameValue and IsSameValueZero? The latter considers -0 and +0 equal but my guess is that isn't obvious from the name (I had to double-check myself)

Reply:
Thanks for your review suggestions.
I use those names from [Equality comparisons and sameness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#same-value-zero_equality) of MDN.
But `JS_IsEqual` and `JS_IsStrictEqual` seems more convenient so I've changed in new commit.